src/schema/v2/filterArtworksConnection.ts

```diff
@@ -643,3 +643,14 @@ export const EntityWithFilterArtworksConnectionInterface = new GraphQLInterfaceT
     },
   }
 )
+
+export const EntityWithArtworksConnectionInterface = new GraphQLInterfaceType({
```

Comment:
To avoid breaking change on `marketingCollections` with `artworksConnection` I had to add this interface, open to better suggestion!